### ISSUE TITLE
Spell list not showing

### ISSUE THREAD

**Issue by dev-wagner-zoccoli** [casualshammy/NameplateCooldowns#110]
**Describe the bug**
The spell selection list is not showing when button is clicked

**To Reproduce**
1. Go to: left panel 'Spells'
2. Click on: right panel 'Click to select spell'
3. List won't open/show
---
4. Scroll down to '....'
5. See error

**Expected behavior**
A list of classes and it's spells should appear with check boxes for specific selection (list not showing)

**Screenshots**

**Comment by dev-wagner-zoccoli** [comment]
Hello! Just reporting.... tks for this addon!

**Comment by casualshammy** [comment]
Hello!
Could you please post a lua error you get? If you don't see any errors, try enter this command to chat: `/console scriptErrors 1`

**Comment by dev-wagner-zoccoli** [comment]
Hi! I tried it again... at first, it kept not showing with no errors. I typed the command you suggested... nothing showed. But then I reported another player, got an error msg and tried again... the list appeared! So I closed the game and did it all ove again and everything seems to be normal. One thing I noticed is that when it didn't work, the button "click to select spell" as grey... now it's red. Well... everything seems to be working now! TY!
